lib/inferno/dsl/fhir_evaluator/rules/abstract_has_examples_rule.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../evaluator_util'
+
+module FhirEvaluator
+  module Rules
+    class HasExamples < Rule
```

Comment:
This class name sounds like a module rather than a class. The filename needs to match the class/module name.

Reply:
The file is deleted.